core/algorithms/simplify.py

```diff
@@ -1306,7 +1307,13 @@ def simplify_pairs(artifacts, roads, distance=2, min_dangle_length=20):
     return roads_cleaned
 
 
-def get_artifacts(roads, deviation=1.01):
+def get_artifacts(
+    roads,
+    area_threshold_blocks=10**5,
+    isoareal_threshold_blocks=0.5,
+    area_threshold_circles=5 * 10**4,
```

Comment:
```suggestion
    area_threshold_blocks=1e5,
    isoareal_threshold_blocks=0.5,
    area_threshold_circles=5e4,
```
we can write these directly without a computation